tasks/backup_transfer.sh

```diff
@@ -18,3 +18,4 @@ if [ "$MANUAL_MIGRATION" = true ]
 fi
 
 rsync -ave 'ssh -o StrictHostKeyChecking=no' "$PT_backupdir"/* root@"$PT_targethost":"$PT_targetdir"
```

Comment:
If we're already running this as a task or ideally part of a plan, can we use [file_upload](https://puppet.com/docs/bolt/latest/plan_functions.html#upload-file)?  Having an `rsync` with a hard-coded `root@` and key checking turned off doesn't seem too useful.